support-frontend/app/actions/UserFromAuthCookiesActionBuilder.scala

```diff
@@ -104,7 +113,7 @@ object UserFromAuthCookiesActionBuilder {
   /** Tries to process a request with the given block. If the request doesn't have valid token cookies, returns a
     * [[ValidationError]].
     */
-  def tryToProcessRequest[A](
+  private def tryToProcessRequest[A](
```

Comment:
The naming of the methods here confuse me a little, although the implementation is very explanatory. I wonder if we could be more explicit about what these methods are doing to help with the clarity of the flow for future travellers.

> [!NOTE]  
> I am new to the codebase, so it could be that I am just unfamiliar with the code

Currently we have:
```mermaid
flowchart LR
    invokeBlock-->processRequest{{processRequest}}
    processRequest-->tryToProcessRequest{{tryToProcessRequest}}
    processRequest-->processRequestWithoutUser

    tryToProcessRequest-->AuthenticatedRequest
    tryToProcessRequest-->handFailure
```


Would something like the below be more explanatory and explicit?
```mermaid
flowchart LR
    invokeBlockA-->shouldValidateUser{{shouldValidateUser}}
    shouldValidateUser-- yes -->validateUserLocally
    shouldValidateUser-- no -->responseWithNoUser
    validateUserLocally-- valid -->responseWithUser
    validateUserLocally-- invalid -->validateUserRemotely
    validateUserRemotely-- valid -->responseWithUser
    validateUserRemotely-- invalid -->responseWithNoUser
```